packages/office-ui-fabric-react/src/components/Pivot/docs/PivotDonts.md

```diff
@@ -1,3 +1,4 @@
 - Don’t use on pages which doesn’t scroll.
 - Don’t use the Pivot to link to a new page.
-- Don’t use the Pivot to link to hidden content.
\ No newline at end of file
+- Don’t use the Pivot to link to hidden content.
+- Don't pass any children other than PivotItems to the Pivot.
```

Comment:
Nit: But can we use the fancy apostrophe `’` here for consistency with the other list items?